src/sdk/main/include/AccountCreateTransaction.h

```diff
@@ -136,7 +136,6 @@ class AccountCreateTransaction : public Transaction<AccountCreateTransaction>
    *
    * @param memo The desired memo for the new account.
    * @return A reference to this AccountCreateTransaction object with the newly-set memo.
-   * @throws std::length_error If the memo is more than 100 characters.
```

Comment:
This should be up to the network to decide if a memo is too long, not the SDK. So this was removed.